packages/expect/src/matchers.js

```diff
@@ -205,134 +165,101 @@ const matchers: MatchersObject = {
 
     const message = pass
       ? () =>
-          matcherHint('.not.toBeInstanceOf', 'value', 'constructor') +
+          matcherHint('.toBeInstanceOf', 'value', 'constructor', {
+            isNot: this.isNot,
+          }) +
           '\n\n' +
-          `Expected value not to be an instance of:\n` +
-          `  ${printExpected(constructor.name || constructor)}\n` +
-          `Received:\n` +
-          `  ${printReceived(received)}\n`
+          `Expected constructor: ${EXPECTED_COLOR(
+            constructor.name || constructor,
+          )}\n` +
+          `Received value: ${printReceived(received)}`
       : () =>
-          matcherHint('.toBeInstanceOf', 'value', 'constructor') +
+          matcherHint('.toBeInstanceOf', 'value', 'constructor', {
+            isNot: this.isNot,
+          }) +
           '\n\n' +
-          `Expected value to be an instance of:\n` +
-          `  ${printExpected(constructor.name || constructor)}\n` +
-          `Received:\n` +
-          `  ${printReceived(received)}\n` +
-          `Constructor:\n` +
-          `  ${printReceived(
+          `Expected constructor: ${EXPECTED_COLOR(
+            constructor.name || constructor,
+          )}\n` +
+          `Received constructor: ${RECEIVED_COLOR(
             received.constructor && received.constructor.name,
-          )}`;
+          )}\n` +
+          `Received value: ${printReceived(received)}`;
```

Comment:
Shouldn't this matcher be collapsed too?

Reply:
Do you mean display all three lines for `not` version of assertion?

If yes, then I agree and will change it to simplify code and make report more consistent.

Reply:
Yup, can be a part of next PR